buildSrc/src/main/kotlin/Java9Modularity.kt

```diff
@@ -0,0 +1,115 @@
+import org.gradle.api.JavaVersion
+import org.gradle.api.Project
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.api.tasks.compile.AbstractCompile
+import org.gradle.api.tasks.compile.JavaCompile
+import org.gradle.kotlin.dsl.attributes
+import org.gradle.kotlin.dsl.findByType
+import org.gradle.kotlin.dsl.getByName
+import org.gradle.kotlin.dsl.register
+import org.gradle.util.GUtil.toCamelCase
+import org.jetbrains.kotlin.gradle.dsl.KotlinProjectExtension
```

Comment:
We prefer star imports in the code style

Reply:
Done